cpp/data/NewsletterUploadForm.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <head>
+    <title>Newsletter Upload Form</title>
+  </head>
+  <body>
+    <form enctype="multipart/form-data" action="/var/www/cgi-bin/newsletter_uploader" method="post">
```

Comment:
- newsletter_uploader CGI program missing
- /var/www/cgi-bin will not work when called via browser

Reply:
https://www.oreilly.com/library/view/cgi-programming-on/9781565921689/07_chapter-04.html